pkgs/top-level/aliases.nix

```diff
@@ -61,6 +61,7 @@ mapAliases ({
   adtool = throw "'adtool' has been removed, as it was broken and unmaintained";
   advcpmv = throw "'advcpmv' has been removed, as it is not being actively maintained and break recent coreutils."; # Added 2024-03-29
   aether = throw "aether has been removed from nixpkgs; upstream unmaintained, security issues"; # Added 2023-10-03
+  afl = throw "aether has been removed as the upstream project was archived. Consider using 'aflplusplus'"; # Added 2024-04-21
```

Comment:
```suggestion
  afl = throw "'afl' has been removed as the upstream project was archived. Consider using 'aflplusplus'"; # Added 2024-04-21
```

Reply:
Whoops i copied the message template :eyes: !

Reply:
done